integration

```diff
@@ -167,6 +167,7 @@
   "Cmajda/ha_golemio",
   "cnstudio/Taipower-Bimonthly-Energy-Cost-homeassistant",
   "CodeFoodPixels/robovac",
+  "codyc1515/ha-meridian-energy",
```

Comment:
The current release offers no documentation at all <https://github.com/codyc1515/ha-meridian-energy/blob/v1.0.1/README.md>